Octokit.Reactive/Clients/ObservableRepositoriesClient.cs

```diff
@@ -650,6 +650,36 @@ public IObservable<Repository> Edit(string owner, string name, RepositoryUpdate
         }
 
         /// <summary>
+        /// Get the contents of a repository's license
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/licenses/#get-the-contents-of-a-repositorys-license">API documentation</a> for more details
+        /// </remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <returns>Returns the contents of the repository's license file, if one is detected.</returns>
+        public IObservable<RepositoryContentLicense> GetLicenseContents(string owner, string name)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
```

Comment:
Can we please update these to the newer style using `nameof(owner)` rather than hardcoded string `"owner"` ?

Reply:
sure